src/frontend/Components/ReactPlayerFrame/ReactPlayerFrame.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import "./ReactPlayerFrame.css";
+import ReactPlayer from "react-player";
+const ReactPlayerFrame = ({ id }) => {
+  return (
+    <div className="react-player-wrapper">
+      <ReactPlayer
+        url={`https://www.youtube.com/watch?v=${id}`}
```

Comment:
Suggestion: You can make util function where you give id and it will give video link after appending all static text.